test/parallel/test-runner-inspect.mjs

```diff
@@ -35,7 +35,6 @@ tmpdir.refresh();
   assert.match(stderr,
                /Warning: Using the inspector with --test forces running at a concurrency of 1\. Use the inspectPort option to run with concurrency/);
   assert.match(stdout, /not ok 1 - .+index\.js/);
-  assert.match(stdout, /stderr: \|-\r?\n\s+Debugger listening on/);
```

Comment:
I suggest calling `subtest.addToReport(ast);`
inside https://github.com/nodejs/node/blob/d97256a32c171e06a9b413e68db5ef12cdec5dbf/lib/internal/test_runner/runner.js#L234-L238
as `TokenKind.UNKNOWN`

Reply:
Do you mean something like this?

```javascript
    child.stderr.pipe(parser).on('data', (ast) => {
      if (ast.lexeme && isInspectorMessage(ast.lexeme)) {
        process.stderr.write(ast.lexeme + '\n');
      }

      if (ast.kind === TokenKind.UNKNOWN) {
        subtest.addToReport(ast);
      }
    });
```

Reply:
yes I might even skip the if

Reply:
Good. I'll push the change tomorrow 👍

Reply:
done.

Reply:
Does the stderr exist somewhere now in one of the tests? If it does, I think the commented out code can be removed (also, where is it now).

Reply:
I am going to revert this change because this is doubling the printed lines (stdout + strerr). Here is an example from `./node --no-warnings --test test/parallel/test-runner-inspect.mjs`

![image](https://user-images.githubusercontent.com/1699357/196122148-c8f94d8c-a806-4a79-a6f0-10959772bbf4.png)

Reply:
This is the change that triggering the doubled output: ![image](https://user-images.githubusercontent.com/1699357/196122684-a3092050-dc5b-4032-8670-363650bdd502.png)